examples/sri-1/components/workgroup/workgroup/workgroupComponent.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Agreement } from '../storage/agreement';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Invitation } from '../invitation/invitation';
+import { Workgroup } from './workgroup';
+import { Workstep } from '../workstep/workstep';
+import { IWorkgroupComponent } from './workgroupComponent.interface';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+
+export class WorkgroupComponent implements IWorkgroupComponent {
+    workgroups: Workgroup[] = [];
+    invitations: Invitation[] = [];
+    archivedWorkgroups: Workgroup[] = [];
+    
+    //TODO Implement checks for actions bound by security and privacy policies
+    getWorkgroups(): Workgroup[] { 
+        return this.workgroups;
+    };
+
+    createWorkgroup(name: string, id: string, owner: BpiSubject, worksteps: Workstep[], securityPolicy: Security, privacyPolicy: Privacy): Workgroup {  //[R236] There MUST be at least one BPI Subject role that has the authorization to create a workgroup. [R237] A workgroup MUST consist of at least one BPI Subject participant.
+        const workgroup = new Workgroup(name, id, owner, worksteps, securityPolicy, privacyPolicy); //TODO Authorize BPI subject to create workgroup #485
```

Comment:
We shall have type check here as well

Reply:
This stand-in implementation will be ultimately replaced as server framework is implemented.